pages/index.tsx

```diff
@@ -0,0 +1,7 @@
+import { LandingPageContainer } from "@features/layout/";
```

Comment:
I'd recommend looking at static site generation (SSG) with Next.js. This page is a perfect use-case for that